components/availability/src/Availability.svelte

```diff
@@ -160,27 +184,32 @@
 
 <nylas-error {id} />
 <main bind:this={main}>
-  {#each days as day}
-    <div class="day">
-      <header>
-        <h2>{new Date(day.timestamp).toLocaleDateString()}</h2>
-      </header>
-      <ul class="slots">
-        {#each day.slots as slot}
-          <li
-            class="slot {slot.status}"
-            on:click={() => {
-              slot.status = handleTimeSlotClick(slot);
-            }}
-            on:mouseover={() =>
-              console.log(new Date(slot.time).toLocaleTimeString())}
-          >
-            <span>{new Date(slot.time).toLocaleString()}</span>
-          </li>
-        {/each}
-      </ul>
-    </div>
-  {/each}
+  <div class="days">
+    {#each days as day}
+      <div class="day">
+        <header>
+          <h2>{new Date(day.timestamp).toLocaleDateString()}</h2>
+        </header>
+        <div class="slots">
+          {#each day.slots as slot}
+            <button
+              class="slot {slot.selectionStatus} {slot.availability}"
+              data-start-time={new Date(slot.start_time).toLocaleString()}
+              data-end-time={new Date(slot.end_time).toLocaleString()}
+              on:mouseover={() =>
+                console.log(
+                  "TODO: temp; ",
+                  new Date(slot.start_time).toLocaleString(),
+                )}
+              on:click={() => {
+                slot.selectionStatus = handleTimeSlotClick(slot);
+              }}
+            />
```

Comment:
Forgot to add: this should have text or an aria-label otherwise it's going to just read out "button" to a user of assistive technology